ETL.md

```diff
@@ -51,16 +59,24 @@ kiba ETL source files are in `lib/etl/`
 
 ```
 ├── common.rb
-├── convert-islandpress.etl
+├── process_local_json_resources.etl
+├── process_local_json_groups.etl
+├── process_s3_json_resources.etl
+├── process_s3_json_groups.etl
 ├── data
-│   ├── sample_island_press_1.epub
-│   └── sample_island_press_2.epub
+│   ├── good_resource_array.json
```

Comment:
Because I think the EPUB parsing as currently is done should be replaced by JSON parsing, I wanted to just reflect that in the ETL.

The reason I didn't actually remove the associated files (sample_island_press_1.epub) in my branch is that I knew we should have this conversation first.